docs/analytics/jenkins.md

```diff
@@ -198,16 +198,17 @@ Also, ensure these repositories are cloned and readable by the jenkins user:
 
 * `/home/jenkins/analytics-configuration`: clone the client's fork and branch, e.g.:
 
-  ```yaml
-  analytics_configuration_repo: 'https://github.com/xxx/edx-analytics-configuration.git'
-  analytics_configuration_version: 'master'
-  ```
+```yaml
+analytics_configuration_repo: 'https://github.com/xxx/edx-analytics-configuration.git'
+analytics_configuration_version: 'master'
+```
+
 * `/home/jenkins/analytics-tasks`: clone the client's fork and branch, e.g.:
 
-  ```yaml
-  analytics_pipeline_repo: 'https://github.com/xxx/edx-analytics-pipeline.git'
-  analytics_pipeline_version: 'master'
-  ```
```

Comment:
Nit: These spaces are required to make the code blocks align with the bullet points